whois-query/src/main/java/net/ripe/db/whois/query/acl/HazelcastPersonalObjectAccounting.java

```diff
@@ -18,14 +18,15 @@
 @Component
 public class HazelcastPersonalObjectAccounting implements PersonalObjectAccounting {
     private static final Logger LOGGER = LoggerFactory.getLogger(HazelcastPersonalObjectAccounting.class);
-
-    private final IMap<InetAddress, Integer> counterMap;
+    private final IMap<InetAddress, Integer> remoteAddrCounterMap;
+    private final IMap<String, Integer> ssoCounterMap;
     private final HazelcastInstance hazelcastInstance;
```

Comment:
We are not using this. It is just composing rest of the variables and the log, so I suggest removing it from the variables and use the parameter of the constructor to compose the rest of the variables

Reply:
It was like this before , so I did not change. Lets do it in separate refactoring Jira to reduce scope. Its alreday a big PR

Reply:
Agree that it is out of scope